noot

{
	const int i1 := 3;
	int i2; // Should throw exception here
};